api/ontology/ontology.proto

```diff
@@ -0,0 +1,640 @@
+// Copyright 2024 Fraunhofer AISEC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//           $$\                           $$\ $$\   $$\
+//           $$ |                          $$ |\__|  $$ |
+//  $$$$$$$\ $$ | $$$$$$\  $$\   $$\  $$$$$$$ |$$\ $$$$$$\    $$$$$$\   $$$$$$\
+// $$  _____|$$ |$$  __$$\ $$ |  $$ |$$  __$$ |$$ |\_$$  _|  $$  __$$\ $$  __$$\
+// $$ /      $$ |$$ /  $$ |$$ |  $$ |$$ /  $$ |$$ |  $$ |    $$ /  $$ |$$ | \__|
+// $$ |      $$ |$$ |  $$ |$$ |  $$ |$$ |  $$ |$$ |  $$ |$$\ $$ |  $$ |$$ |
+// \$$$$$$\  $$ |\$$$$$   |\$$$$$   |\$$$$$$  |$$ |  \$$$   |\$$$$$   |$$ |
+//  \_______|\__| \______/  \______/  \_______|\__|   \____/  \______/ \__|
+//
+// This file is part of Clouditor Community Edition.
+
+
+syntax = "proto3";
+
+package clouditor.ontology.v1;
+
+import "google/api/annotations.proto";
+import "google/protobuf/struct.proto";
+import "tagger/tagger.proto";
+import "validate/validate.proto";
+
+option go_package = "clouditor.io/clouditor/api/ontology";
+
+message ResourceID {
+	repeated string resource_id = 1;
+}
+
+// Analyzes the activity of a NetworkService (which includes DatabaseServices).
+// Scope contains the resource ID of the protected resource.
+message AnomalyDetection {
+	bool enabled  = 1;
+	ApplicationLogging ApplicationLogging  = 2;
+}
+
+message Networking {
+
+	oneof type {
+		NetworkInterface network_interface = 101;
+		NetworkSecurityGroup network_security_group = 102;
+		NetworkService network_service = 103;
+		VirtualNetwork virtual_network = 104;
+		VirtualSubNetwork virtual_sub_network = 105;
+	}
+}
+
+message CustomerKeyEncryption {
+	string keyUrl  = 1;
+}
+
+message DatabaseQuery {
+	bool modify  = 1;
+}
+
+// An HTTP endpoint, that is routed through a (reverse) proxy, e.g. a load balancer.
+message ProxiedEndpoint {
```

Comment:
I thin we made a little hack in the owl2java, that the description starts with `((Entity is)) ...` so that in this case the description would be `ProxiedEndpoitn is an HTTP endpoint` to generate go compliant docs.

Reply:
Done in owl2Protobuf